kolibri_explore_plugin/locale/es_419/LC_MESSAGES/ek-components-messages.json

```diff
@@ -0,0 +1,38 @@
+{
+  "EkCardButtons.downloadCompletedLabel": "Descarga completa",
+  "EkCardButtons.downloadLabel": "Descargar",
+  "EkCardButtons.downloadRestartLabel": "Reiniciar descarga",
+  "EkCardButtons.downloadingLabel": "Descargando…",
+  "EkCollapsibleCardGrid.showLessButton": "Mostrar menos",
+  "EkCollapsibleCardGrid.showMoreButton": "Mostra más",
+  "EkCollapsibleCardGrid.spinnerLabel": "Cargando",
+  "EkComponentsConstants.artistSubtitle": "Soy creativo y disfruto haciendo cosas, de la música y de bailar. ",
+  "EkComponentsConstants.artistTitle": "Artista",
+  "EkComponentsConstants.athleteSubtitle": "Amo mover mi cuerpo y estar con mis amigos. ",
+  "EkComponentsConstants.athleteTitle": "Atleta",
+  "EkComponentsConstants.curiousSubtitle": "Me gusta experimentar un poco de todo. ",
+  "EkComponentsConstants.curiousTitle": "Curioso",
+  "EkComponentsConstants.discoverVerb": "Descubre",
+  "EkComponentsConstants.explorerSubtitle": "Me gusta aprender de diferentes culturas, lugares e ideas. ",
+  "EkComponentsConstants.explorerTitle": "Explorador",
+  "EkComponentsConstants.interactVerb": "Interactúa",
+  "EkComponentsConstants.inventorSubtitle": "Me gusta construir cosas y resolver problemas. ",
+  "EkComponentsConstants.inventorTitle": "Inventor",
+  "EkComponentsConstants.listenVerb": "Escucha",
+  "EkComponentsConstants.practiceVerb": "Practica",
+  "EkComponentsConstants.readVerb": "Lee",
```

Comment:
Yeah, it was expected that there would have to be a few cleanup passes of marking more strings as translatable, or reworking how some strings are handled so the code is more translatable.

Perhaps split this out to a follow-up issue and I’ll take a look at it when I can next pick up a task (if someone else doesn’t get there first).

Reply:
hmm, template-ui and ek-components translations were definitely working last week when I tested them. This is definitely material for a follow-up issue, otherwise this PR is going to bitrot fast.

Reply:
My guess is that (currently) they need to be translated in 2 contexts:

* In `kolibri_explore_plugin.app` so that ek-components and template-ui translations show up in the main plugin
* Standalone so they work from the channel override web apps

Reply:
Kolibri has a somewhat equivalent setup with [kolibri-common](https://github.com/learningequality/kolibri/tree/release-v0.16.x/packages/kolibri-common). It's a library of components shared between their plugins. Maybe we can glean some useful information from how that's used.

Reply:
I’m looking at this

Reply:
> * Standalone so they work from the channel override web apps

The channel override webapps are using the translations through KolibriApi, so the translations need to be in `kolibri_explore_plugin.app`

Reply:
I’ve split this out into a separate issue, https://github.com/endlessm/kolibri-explore-plugin/issues/846, otherwise it’s going to be hard to track.